src/wrapper/result_.mli

```diff
@@ -5,9 +5,15 @@ type t
 
 val create : unit -> t
 val column_count : t -> int
+val column_name : t -> int -> string
+val column_type : t -> int -> Type.t
```

Comment:
Do these two functions actually need to be exposed? Why not extract the relevant information via a call to `Result_.schema` instead?

Reply:
I don't think you've addressed this issue.